core/quorum_initiator.go

```diff
@@ -150,6 +155,7 @@ func (c *Core) QuroumSetup() {
 	c.l.AddRoute(APIUpdatePledgeToken, "POST", c.updatePledgeToken)
 	c.l.AddRoute(APISignatureRequest, "POST", c.signatureRequest)
 	c.l.AddRoute(APISendReceiverToken, "POST", c.updateReceiverToken)
+	c.l.AddRoute(APIUnlockTokens, "POST", c.unlockTokens)
```

Comment:
Will this unlock all the locked token or just the token locked in this specific pledge request?

Reply:
It will unlock the tokens for the specific pledge request.